test-cases/PR-provision-test.yaml

```diff
@@ -0,0 +1,21 @@
+test_duration: 10
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=1m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=1m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=5 -pop seq=1..10000000"
+             ]
+
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -max-rate=300 -replication-factor=3 -partition-count=10 -clustering-row-count=100 -clustering-row-size=5120 -concurrency=7 -rows-per-request=10",
+                     "scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=10 -clustering-row-count=100 -clustering-row-size=5120 -rows-per-request=10 -concurrency=7 -max-rate=32000 -duration=1m"
+                    ]
+
+n_loaders: 1
+instance_type_db: 'i3.large'
+n_monitor_nodes: 1
+n_db_nodes: 3
+
+nemesis_class_name: ChaosMonkey
+nemesis_interval: 1
+
+user_prefix: 'PR-provision-test'
+instance_provision: 'spot_low_price'
+
+scylla_version: 3.1.0.rc2
```

Comment:
Why do you need to provide scylla_version?
Do you want to run it on a fixed version?

Reply:
we are not checking Scylla but the stability of SCT code. We prefer to keep it constant.